src/DotNetLightning.Core/Utils/LNMoney.fs

```diff
@@ -91,7 +98,7 @@ type LNMoney = | LNMoney of int64 with
     static member inline (*) (LNMoney a, b) = LNMoney(a * (int64 b))
     static member Max(LNMoney a, LNMoney b) = if a >= b then LNMoney a else LNMoney b
     static member Min(LNMoney a, LNMoney b) = if a <= b then LNMoney a else LNMoney b
-    
+
```

Comment:
nit: no need for this hunk :P (better leave formatting changes to automatic formatting tools, like I did in PR 188 (NB: those tools already remove trailing whitespace automatically)